.github/workflows/nightly.yml

```diff
@@ -72,6 +72,9 @@ jobs:
           command: build
           args: --all-targets --all-features --release
 
+  examples:
```

Comment:
Please add this to the Confluence page

Reply:
Done https://iotafoundation.atlassian.net/wiki/spaces/RD1/pages/2233401375/CI+Architecture